composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "08fdc196dedea64b0e5f907e24e52446",
+    "content-hash": "4105e60ad0aa71a672c8dacd01b6d30c",
```

Comment:
try running `composer install --no-dev` before committing. It will prevent the dev dependencies from being included.